src/routes/articleRoutes.ts

```diff
@@ -0,0 +1,181 @@
+import express from "express";
+import type {Request, Response} from "express";
+import auth from '../../src/middlewares/auth.js'
+import prisma from '../lib/prisma.js'
+
+const router = express.Router();
+
+// [POST] 게시글 등록
+router.post("/", auth, async (req: Request, res: Response): Promise<Response> => {
+  try {
+    const { title, content } = req.body as { title?: string; content?: string };
+
+    if (!title || !content) {
+      return res.status(400).json({ message: "제목과 내용을 모두 입력해주세요." });
+    }
+
+    const newArticle = await prisma.article.create({
+      data: { title, content },
+    });
```

Comment:
```suggestion
    const newArticle = await prisma.article.create({
      data: { title, content, authorId: res.locals.user.userId },
    });
```
위와 같은 방식으로 인가 기능도 추가해줘야겠죠?